quiz.js

```diff
@@ -71,12 +79,27 @@ const questions = [
       <p>${titleText}</p>
       <p>${contentText}</p>
     `;
+    lastButton.style.display = 'block';
+ 
   }
 
   function startQuiz(){
+     const mainElement = document.querySelector('main');
+    mainElement.innerHTML= `
+    <p>treść pytania</p>
+    <radiogroup class="answers">         
+        <label id="answer1_label"><input type="radio" id="answer1" name="answer" value="a"/>odpowiedź pierwsza</label>
+        <label id="answer2_label"><input type="radio" id="answer2" name="answer" value="b"/>odpowiedź druga</label>
+        <label id="answer3_label"></label><input type="radio" id="answer3" name="answer" value="c"/>odpowiedź trzecia</label>
+    </radiogroup>
+    `
```

Comment:
świetnie, że pomyślałaś o dostosowaniu tej funkcji, żeby działała poprawnie po Twoich zmianach 👍 . Tak, tu po prostu musimy wkleić zawartość maina taką, jak w html. Index.html się zmienił, bo ma teraz przycisk "Dalej" w main, więc zajrzyj tam teraz. Jedyne co, to zawartość radiogroup (te wszystkie trzy inputy) możesz usunąć, bo funkcja showAnswers powinna dodawać je dynamicznie (mam nadzieję, że tak będzie). Wtedy w index.html też wyczyścimy tę część więc ty możesz to zrobić już teraz w funkcji startQuiz.